src/main/java/seedu/address/model/borrower/Email.java

```diff
@@ -23,9 +23,10 @@
     private static final String LOCAL_PART_REGEX = "^[\\w" + SPECIAL_CHARACTERS + "]+";
     private static final String DOMAIN_FIRST_CHARACTER_REGEX = "[^\\W_]"; // alphanumeric characters except underscore
     private static final String DOMAIN_MIDDLE_REGEX = "[a-zA-Z0-9.-]*"; // alphanumeric, period and hyphen
-    private static final String DOMAIN_LAST_CHARACTER_REGEX = "[^\\W_]$";
+    private static final String DOMAIN_LAST_CHARACTER_REGEX = "[^\\W_]{2,}$";
     private static final String VALIDATION_REGEX = LOCAL_PART_REGEX + "@"
```

Comment:
Beautiful regex 💯

Reply:
> Beautiful regex 💯

Actually spent some time understanding and getting my brain fried